app/src/main/res/layout/list_medical_history_question.xml

```diff
@@ -29,47 +30,29 @@
       android:textColor="?attr/colorOnSurface"
       tools:text="Kidney disease" />
 
-    <LinearLayout
+    <com.google.android.material.chip.ChipGroup
```

Comment:
One thing which I believe we used to do earlier was set the layout such that the width of the "Yes/No" widget would be consistent regardless of whether it was checked or not. What I see with the new chip widgets is that the width changes a lot when checking and unchecking the buttons. Would you be able to check this?

Reply:
There will be some width change, according to the mockups and the normal component. But the reason why it seems too much is probably because of the 12dp start and end padding I have applied. I have to change it to 8dp and it seems fine.